compiler/src/iree/compiler/Dialect/Stream/Transforms/UnifyEncodingForGlobals.cpp

```diff
@@ -88,16 +116,28 @@ class GlobalEncodingAnalyzer {
     return result;
   }
 
-  // Returns the SourceGlobalInfo for the given source global name, or
+  // Returns the SourceGlobalInfo for the given source global name. There is a
+  // copy in the call, so it is not a cheap call.
+  SourceGlobalInfo getSourceGlobals(StringRef name) {
+    assert(sourceGlobals.contains(name) &&
+           "Expected all the source globals are parsed");
+    return sourceGlobals.find(name)->second;
```

Comment:
```suggestion
    return sourceGlobals.at(name);
```
`::at()` aborts (i.e., `assert`s) if the entry doesn't exist.